ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java

```diff
@@ -118,16 +120,16 @@ public ImportCRFDataService(DataSource ds, Locale locale) {
                 for (FormDataBean formDataBean : formDataBeans) {
 
                     CRFVersionDAO crfVersionDAO = new CRFVersionDAO(ds);
+                	FormLayoutDAO fldao= new FormLayoutDAO<>(ds);

```

Comment:
Do we need to create the DAO object in the loop or we can reuse it by creating it only once?